lib/core/index.js

```diff
@@ -1,9 +1,12 @@
 var Analytics = require('./analytics');
+var fs = require('fs');
+var path = require('path');
 
 var analytics = new Analytics();
 
 analytics.require = require;
 
 analytics.VERSION = require('../../package.json').version;
+analytics.AGREEMENT = fs.readFileSync(path.join(__dirname, '../../AGREEMENT'), 'utf-8');
```

Comment:
Can we inline the agreement here? Doing `fs.readFileSync` will not be possible on the browser :(

Reply:
hmm... right!